src/codecs/types.ts

```diff
@@ -7,11 +7,14 @@ export interface GlobalEncodeOptions<CodecOptions extends object = {}> {
   filePath?: string
   isStandalone?: boolean
   isBundle?: boolean
+  shouldZip?: 'yes' | 'no' | 'maybe'
```

Comment:
I don't know? Can you repeat the question?

https://www.youtube.com/watch?v=J8y8Ziwswe0